.github/workflows/svgo.yml

```diff
@@ -19,18 +19,22 @@ jobs:
       - name: Set up node
         uses: actions/setup-node@v4
 
-      - name: Install SVGO
-        run: npm install --global svgo
-
       - name: Compress SVG assets and commit to current PR
         run: |
-          svgo --multipass --final-newline assets --exclude ptable-scatters-*.svg
+          # Install SVGO
+          npm install --global svgo
+
+          # Show SVGO version (DEBUG #187)
+          # TODO: remove after 2025-01-01 if nothing is going wrong
+          svgo --version
+
+          svgo --multipass --final-newline assets
           if git diff --quiet assets; then
             echo "No changes to commit"
             exit 0
           fi
           git config user.name "Janosh Riebesell"
           git config user.email janosh.riebesell@gmail.com
           git add assets
-          git commit -m "compress new SVG assets"
+          git commit -m "[workflow] Compress new SVG assets"
```

Comment:
Added this prefix to distinguish between a commit added by automatic workflow (I thought the original commit was done by you yourself)

Reply:
good idea